src/compiler/watch.ts

```diff
@@ -73,7 +73,7 @@ namespace ts {
                     output += newLine;
                 }
 
-                output += `${new Date().toLocaleTimeString()} - `;
+                output += `${new Date().toLocaleTimeString("en-US", { hour12: false })} - `;

```

Comment:
The idea was to localize the time based on the user, not to always use a 24 hour clock.